packages/reducers/app.ts

```diff
@@ -157,3 +161,35 @@ export const calculateCounterparties = (state) => () : number => {
 
   return Object.keys(friends).length - 1
 }
+
+export const calculateUcacBalances = (state) => (friendAddress: string) : Object => {
+  const recent = recentTransactions(state)
```

Comment:
I'd prefer to call this 'recents' since it's plural, i.e. an array of things. If you do change it, change it in the previous function as well.